docs/supported_languages.rst

```diff
@@ -0,0 +1,125 @@
+Policy on Supported Languages
+=============================
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Version
+     - Approved
+   * - `5 <https://github.com/freedomofpress/securedrop-engineering/issues/6>`_
+     - 8 March 2023
+
+.. note::
+   The key words *MUST*, *MUST NOT*, *REQUIRED*, *SHALL*, *SHALL NOT*, *SHOULD*,
+   *SHOULD NOT*, *RECOMMENDED*,  *MAY*, and *OPTIONAL* in this document are to be
+   interpreted as described in `RFC 2119`_.
+
+Thresholds for Translation and Review Coverage
+----------------------------------------------
+
+.. list-table::
+   :widths: 30 30 30
+   :header-rows: 1
+
+   * -
+     - Translation Coverage (of Source Strings)
+     - Review Coverage (of Translations) [#review_coverage]_
+   * - **Initial coverage,** for a language to be granted support
+     - 100%
+     - 100%
+   * - **Ongoing coverage,** for a language to remain supported
+     - 80%
+     - 100%
```

Comment:
I would suggest a simpler format, as the table currently requires horizontal scrolling, which is not obvious until you mouse over it (at least on my browser/OS combo).
![Screenshot from 2023-03-14 17-23-29](https://user-images.githubusercontent.com/213636/225171696-efb32646-a719-4c97-bfc9-ea83b7cfb03d.png)

Reply:
I've simplified the tabular format, which feels to me like what presentation this information "wants", without much loss of information, in 7d7cff1e6c981fee00288c649ffedf7f09ac8645.  Does that do the trick? or would you really prefer a list to this table?

Reply:
Looks great!